examples/acl/src/backend/app/views/posts.clj

```diff
@@ -0,0 +1,25 @@
+(ns views.posts
+  (:require
+    [xiana.core :as xiana]))
+
+(defn post-view
+  [{response :response-data :as state}]
+  (xiana/ok
+    (assoc state
+      :response
+      {:status  200
+       :headers {"Content-Type" "text/plain"}
+       :body    (str "Place of one post\n" response)})))
+
+(defn all-posts
+  [{response :response-data :as state}]
+  (xiana/ok
+    (assoc state
+      :response
+      {:status  200
+       :headers {"Content-Type" "text/plain"}
+       :body    (str "Place for all posts, you are able to see: " response)})))
```

Comment:
Such description should be in docs, like swagger, and not returned with each endpoint call